src/components/map-page-components/map-view/MapView.tsx

```diff
@@ -1,7 +1,203 @@
 import React from 'react';
+import { Input } from 'antd';
+import { Loader } from '@googlemaps/js-api-loader';
+import styled from 'styled-components';
+
+const { Search } = Input;
+
+const StyledSearch = styled(Search)`
+  width: 40vw;
+  position: absolute;
+  z-index: 2;
+`;
+
+let map: google.maps.Map;
+
+const BOSTON: google.maps.LatLngLiteral = { lat: 42.315, lng: -71.0589 };
+const BOSTON_BOUNDS = {
+  north: 42.42,
+  south: 42.2,
+  west: -71.28,
+  east: -70.83,
+};
+
+const loader = new Loader({
+  apiKey: process.env.REACT_APP_GOOGLE_MAPS_KEY!,
+  libraries: ['places'],
+  mapIds: ['76c08a2450c223d9'],
+});
+
+loader.load().then(() => {
+  map = new google.maps.Map(document.getElementById('map') as HTMLElement, {
+    center: BOSTON,
+    zoom: 12,
+    fullscreenControl: false,
+    mapTypeControl: false,
+    // mapId: 76c08a2450c223d9,
+    restriction: {
+      latLngBounds: BOSTON_BOUNDS,
+      strictBounds: false,
+    },
+  });
+
+  const input = document.getElementById('pac-input') as HTMLInputElement;
+  const autocomplete = new google.maps.places.Autocomplete(input);
+  autocomplete.addListener('place_changed', () => {
+    const place = autocomplete.getPlace();
+
+    if (!place.geometry) {
+      window.alert("No details available for input: '" + place.name + "'");
+      return;
+    }
+
+    // If the place has a geometry, then present it on a map.
+    if (place.geometry.viewport) {
+      map.fitBounds(place.geometry.viewport);
+    } else {
+      map.setCenter(place.geometry.location);
+      map.setZoom(16);
+    }
+  });
+
+  const privateStreetsLayer = new google.maps.Data({ map });
+
+  privateStreetsLayer.loadGeoJson(
+    'https://raw.githubusercontent.com/florisdobber/SFTT-map-test/master/private_streets.json',
+  );
+
+  function setPrivateStreetsStyle(v: boolean) {
+    privateStreetsLayer.setStyle({
+      strokeColor: 'red',
+      strokeWeight: 2,
+      visible: v,
+    });
+  }
+
+  setPrivateStreetsStyle(false);
+
+  const blocksLayer = new google.maps.Data({ map });
+  blocksLayer.loadGeoJson(
+    'https://raw.githubusercontent.com/florisdobber/SFTT-map-test/master/blocks.json',
+  );
+
+  function setBlocksStyle(v: boolean) {
+    blocksLayer.setStyle((feature) => {
+      let color = 'green';
+
+      // Use this for coloring reserved/completed blocks a different color
+      if (feature.getProperty('ID') % 10 === 0) {
+        color = 'yellow';
+      }
+
+      if (feature.getProperty('ID') % 10 === 1) {
+        color = 'red';
+      }
+
+      // Use this for selecting blocks
+      if (feature.getProperty('PRECINCT') === 'YES') {
+        color = 'black';
+      }
```

Comment:
That's a sample for how to select blocks. Will be used to reserve blocks once that is up. Since I had already figured out how to do it seemed easier to leave it in the code and all someone has to do is adjusted the code.